compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/CodeGenerator.java

```diff
@@ -52,18 +63,31 @@
 public class CodeGenerator {
 
     private static final CompilerContext.Key<CodeGenerator> CODE_GEN = new CompilerContext.Key<>();
-
     private SymbolTable symbolTable;
-
-    private Map<String, BIRNode.BIRPackage> compiledPkgCache = new HashMap<>();
-
-    private JvmPackageGen jvmPackageGen;
-
-    private CodeGenerator(CompilerContext context) {
-
-        context.put(CODE_GEN, this);
-        symbolTable = SymbolTable.getInstance(context);
-        jvmPackageGen = JvmPackageGen.getInstance(context);
+    private PackageCache packageCache;
+    private BLangDiagnosticLogHelper dlog;
+    private BIREmitter birEmitter;
+    private boolean isBaloGen;
```

Comment:
Shall we rename to `baloGen` or `baloGenerated` and remove `is` prefix to be consistent with other boolean variable names
```suggestion
    private boolean baloGen;
```

Reply:
ack